### PR TITLE
Show the repo count in code tab on both user profile and org page.

### PR DESCRIPTION
**Pull request by puni9869**
as title

Screenshot before
User Profile page 
image

after 
image

Org page
image

### CODE REVIEW

File: routers/web/user/code.go [hunk 1]
```diff
@@ -27,6 +27,16 @@ func CodeSearch(ctx *context.Context) {
...
 
+	var (
+		repoIDs []int64
+		err     error
+	)
+	err = shared_user.LoadHeaderCount(ctx)
+	if err != nil {
+		ctx.ServerError("LoadHeaderCount", err)
+		return
+	}
```
Comment (wxiaoguang) [inline routers/web/user/code.go]:
```suggestion
	if err := shared_user.LoadHeaderCount(ctx); err != nil {
		ctx.ServerError("LoadHeaderCount", err)
		return
	}
```


Then old code doesn't need to change.

Comment (puni9869) [reply routers/web/user/code.go]: Done

**Comment by puni9869** [comment]
Ready for review

**Review by wxiaoguang — APPROVED** [review]

**Comment by puni9869** [comment]
@denyskon  Could you approve this. If you are around.

**Comment by puni9869** [comment]
Need one more approval to merge. 
🚀

**Review by denyskon — APPROVED** [review]

**Comment by puni9869** [comment]
🚀

**Comment by puni9869** [comment]
@silverwind  @lunny  could you send the backport on my behalf, I have not sufficient permissions.

**Pull request merged**